### ISSUE TITLE
Problem when reproducing

### ISSUE THREAD

**Issue by will-wang19** [qiqiApink/MotionGPT#15]
Hello, I had a little problem reproducing this paper. When I tried to finetune the llama 7B on humanml_3d dataset using the funetune_motion.py, it didn't exhibit the performance described in the paper. But when I finetune the llama 7B on kit dataset, it nearly matched the results in the paper. This confused me a lot. I'm looking forward to your reply.

**Comment by qiqiApink** [comment]
Can you show me the performance?

**Comment by ruilin19** [comment]
Sorry, I didn't use the same mean and std files to process the data, after I changed it, the performance nearly matched the results in the paper.

**Comment by GuocunWang** [comment]
Hi, did you modify any hyperparameters when you reproduced the results of llama7B in the T2M task? For example, learning rate, etc. I only changed 13B to 7B in the script and used the original hyperparameters, the training of the model could not converge and could not reproduce the results of the paper. @ruilin19
